### ISSUE TITLE
Single-line Documentation considered confusing

### ISSUE THREAD

**Issue by manveru** [naqvis/crystar#18]
See for example: https://naqvis.github.io/crystar/Crystar/Header.html
![20230823_11h32m54s_grim](https://github.com/naqvis/crystar/assets/3507/9bbd2d46-abc2-4730-afdd-ebe71ddfc259)
A lot of these methods have the wrong documentation because the comment is at the end of the line instead of the previous one.

**Comment by naqvis** [comment]
Agreed. comments was there for code comments purposes, but code formatting made them doc comments. Any PR fixing or updating them is highly appreciated.
